src/core/thread/mle.cpp

```diff
@@ -637,6 +641,17 @@ void Mle::SetAssignLinkQuality(const Mac::ExtAddress aMacAddr, uint8_t aLinkQual
     }
 }
 
+uint8_t Mle::GetRouterSelectionJitter(void) const
+{
+    return mSetRouterJitter ? mRouterJitter : mRouterSelectionJitterTimeout;
+}
+
+void Mle::SetRouterSelectionJitter(uint8_t aRouterJitter)
+{
+    mSetRouterJitter = true;
```

Comment:
Wouldn't it be better to have a variable `mRouterSelectionJitter` and initialize it to `kRouterSelectionJitter` in the constructor?  Then whenever you compute a new jitter value, you just use `otPlatRandomGet() % mRouterSelectionJitter`.  With a variable, you can still tell whether or not the value was changed in the CLI by comparing it against `kRouterSelectionJitter` constant.